packages/frontend/src/providers/ExplorerProvider.tsx

```diff
@@ -102,6 +102,19 @@ const defaultState: ExplorerReduceState = {
     limit: 500,
 };
 
+const calcColumnOrder = (
```

Comment:
This is an effort to prevent us sending the columns order to the BE without some of the selected fields, which would save the missing fields with the order `-1`.

Guaranties we don't blindly trust the url params on page load ( old urls before this feature ) and update the order when we add or remove any field.